### DISCUSSION TITLE
appWithTranslation with Next 13.4 / App Router

### DISCUSSION THREAD

**Discussion by joaopbnogueira** [i18next/next-i18next#2162] [Q&A] [answered]
With the stable release of the App Router in next 13.4, it's unclear how we should inject the HOC appWithTranslation since there's no longer an __app.tsx file.

Could anyone give me a few pointers?

Thank you

**Comment by adrai** [comment, marked as answer]
https://github.com/i18next/next-i18next/issues/2158#issuecomment-1556205214
https://github.com/i18next/next-i18next/discussions/1993#discussioncomment-4337519

**Comment by joaopbnogueira** [comment]
Thank you, I missed thos!
